tests/integration/record_benchmark.py

```diff
@@ -46,55 +51,61 @@
 
 data = {}
```

Comment:
If we are using local `data`, maybe move this declaration down near the `if __name__ == "__main__"` part